admin/services/data-access/check.data.service.js

```diff
@@ -333,4 +333,29 @@ checkDataService.sqlUpdate = async function (checkData) {
   return sqlService.update('[check]', checkData)
 }
 
+checkDataService.sqlFindAllFormsUsedByPupils = async function (pupilIds) {
+  const select = `SELECT 
+    f.id,
+    f.name,
+    c.pupil_id     
+  FROM ${sqlService.adminSchema}.${table} c JOIN
```

Comment:
better to be explicit and specify `INNER JOIN` if that is your intention?

Reply:
`Join` defaults to `INNER` on MS-SQL but no problem being explicit about it.  Pushed.